src/cli/backup/exchange.go

```diff
@@ -272,20 +273,23 @@ func createExchangeCmd(cmd *cobra.Command, args []string) error {
 
 	sel := exchangeBackupCreateSelectors(user, exchangeData)
 
-	users, err := m365.UserPNs(ctx, acct)
+	// TODO: log/print recoverable errors
+	errs := fault.New(false)
```

Comment:
Should this error be involved in the logic in line 279?

Reply:
It's not entirely what its purpose is when there is `merrs` and `ferrs`.

Reply:
`merrs` will eventually go away.  The cli refactor is going to be the last piece of this migration, so if you see code in a halfway state like this, it's pending total completion, which is why we have both.